controllers/fetchSelected.js

```diff
@@ -0,0 +1,12 @@
+const Property = require('../models/Property');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
+  Property.findById(id)
+    .then(property => {
+      res.send(property);
+    })
+    .catch(err => {
+      res.status(400).send('unable to read from the database');
```

Comment:
this should be status code 500 

> 500’s: Server error codes indicating that the request was accepted, but that an error on the server prevented the fulfillment of the request.

Reply:
Why don't you just request all changes once.

Reply:
Look at the entire code and tell me what to change, once and for all.

Reply:
@hoslack I know it's frustrating but sometimes we miss things the first time too! We're having to review so many pull requests for these projects, please bear with us if we miss things.

Reply:
@claireinez I am not frustrated at all. I just feel like letting Mario finish with the whole thing since now I am working on a different branch and wouldn't want to be moving back and forth the branches. I am still not so great with that kind of workflow.

Reply:
Git is confusing for everyone! 😄